brayns/io/MorphologyLoader.cpp

```diff
@@ -25,7 +25,7 @@
 #include <brayns/common/geometry/Cylinder.h>
 #include <brayns/common/geometry/Cone.h>
 #include <brayns/common/scene/Scene.h>
-#include <brayns/common/simulation/SimulationDescriptor.h>
+#include <brayns/io/simulation/SimulationDescriptor.h>
```

Comment:
Since they are used but the Scene class, simulation related classes must stay in the common lib, not in the IO.